src/components/TribeMember/TribeMemberBack.js

```diff
@@ -13,6 +13,7 @@ const TribeMemberBack = ({
   bio,
   social,
   preventTabFocus,
+  display,
```

Comment:
Is this in use?

Reply:
No - thank you I completely missed that! Will remove that!